src/Modules/OrchardCore.Commerce/Views/BooleanProductAttributeField.cshtml

```diff
@@ -7,7 +7,7 @@
     var settings = attr.Settings as BooleanProductAttributeFieldSettings;
     var id = $"{Model.IdPrefix}-attribute-{index}";
 }
-
+<input type="hidden" name="line.Attributes[@index].Value" value="false" />
```

Comment:
Ended up restoring this as without it the attribute does not show up at all due to the missing value.

Reply:
It was causing issues, but then it wasn't and now it isn't. It's weird. UI tests are passing so I suppose it's fine?